### ISSUE TITLE
Dynamically Setting The Grid Height

### ISSUE THREAD

**Issue by simonv3** [TheTorProject/ooni-api#11]
Apparently this isn't quite as easy as it seems:

https://github.com/angular-ui/ui-grid/issues/1735

I'm sure we can figure something out - I think not scrolling on the grid would be the best way to present it, but I was wondering what other people think.

**Comment by hellais** [comment]
Yes I agree that it's not optimal to have the grid be fixed in height and having to scroll inside of it. The pull request mentioned inside of the above ticket seems promising, perhaps we can try to include that patch inside of our codebase and use that?

**Comment by simonv3** [comment]
Looks like that PR was finally merged. Worth checking out before rewriting the grid? @bnvk @hellais

**Comment by krystian-3c** [comment]
+1
